react-app/src/components/Common/clinkmenu.js

```diff
@@ -70,7 +70,7 @@ class ClinkMenu extends React.Component {
 
     return (
       <>
-        <Dropdown overlay={menu} trigger={['click']}>
+        <Dropdown overlay={menu} trigger={['click']} className={this.props.menuClass}>
```

Comment:
I noticed that in fetch-users that clickmenu.js was refactored into a function (it's no longer a class). I don't think you will be able to use this.props.menuClass but you can still passing in a prop. I would suggest merging this after fetch-users so you can pass in the prop another way.

Reply:
Yep we should really merge in Alisa's change.